tidals/clean.py

```diff
@@ -63,3 +64,46 @@ def round_time(df, timeIntervalMinutes=5, timeField="time",
         df.drop(columns=["TIB", "TIB_cumsum"], inplace=True)
 
     return df
+
+
+def remove_brackets(df, fieldName):
+    if fieldName in list(df):
+        df.loc[df[fieldName].notnull(), fieldName] = \
+            df.loc[df[fieldName].notnull(), fieldName].str[0]
+
+    return df
+
+
+def flatten_json(df):
+
+#    # remove fields that we don't want to flatten
+#    df, holdData = tempRemoveFields(df)
```

Comment:
Why are you commenting out this code? And if you do mean to get rid of it, why not just delete it? Same goes further down.

Reply:
good catch...thanks...I have removed the commented code